gulpfile.js

```diff
@@ -62,7 +66,24 @@ function generateScript(inFile, outFile, minify, callback, format) {
         .pipe(rename(outFile)),
   ];
   if (minify) {
-    parts.push(uglify());
+    if (format === 'es') {
+      parts.push(terser({
+        compress: {
+          module: true
+        },
+        ecma: 5, // specify one of: 5, 6, 7 or 8
+        keep_classnames: false,
+        keep_fnames: false,
+        ie8: false,
+        module: false,
```

Comment:
false? doesn't that mean that this will produce es2015 bundle rather than esm2015? in which case all the "esm2015" references should really be "es2015" referenfes. no?

Reply:
I will check it again!

Reply:
This option in Terser reads:
```
module (default false) — Use when minifying an ES6 module. "use strict" is implied and names can be mangled on the top scope. If compress or mangle is enabled then the toplevel option will be enabled.
```

Setting this option to `true` is helpful when minifying application code because you are not concerned with exposed public APIs and want the smallest possible self-contained application. But for libraries, you do not want to either mangle or remove code top level code.

So this option should be set to false, or left to default.

Please bear in mind that Terser is not capable of altering the bundle format. Terser options such as `ecma`, `toplevel` and `module` are meant only to enable/disable available optimizations that are applicable only in these cases. But the module format is always left unchanged.

Reply:
@filipesilva, got it, I will remove those options and test again.